app/templates/main/browse-all.html

```diff
@@ -69,10 +70,10 @@
                             </tr>
                             <tr class="govuk-table__row browse__table__all_desktop__row">
                                 <td class="govuk-table__cell browse__table__mobile--hidden browse__table__all_desktop">
-                                    <a href="{{ url_for('main.browse_transferring_body', _id=record['transferring_body_id']) }}">{{ record["transferring_body"] }}</a>
+                                    <a href="{{ url_for('main.browse_transferring_body', _id=record['transferring_body_id'], _anchor='browse-records') }}">{{ record["transferring_body"] }}</a>
```

Comment:
do we want to go to the browse-records section in these scenarios, do we not just want to do it when applying or clearing filters?

Reply:
Checked with Terry on the UX for this as I wasn't sure myself. It should be applied for:

- Links within browse table
- Applied filters
- Cleared filters

Reply:
sweet 👍